sdk/keyvault/Azure.Security.KeyVault.Shared/tests/KeyVaultTestEnvironment.cs

```diff
@@ -12,5 +12,13 @@ public KeyVaultTestEnvironment() : base("keyvault")
         }
 
         public string KeyVaultUrl => GetRecordedVariable("AZURE_KEYVAULT_URL");
+
+        /// <summary>
+        /// Gets the value of the "KEYVAULT_SKU" variable, or "premium" if not defined.
+        /// </summary>
+        /// <remarks>
+        /// Test preparation was previously successfully creating premium SKUs (not available in every cloud), so assume premium.
+        /// </remarks>
+        public string Sku => GetRecordedOptionalVariable("SKU") ?? "premium";
```

Comment:
Does it have to be recorded?

Reply:
I suppose not, but that does raise an issue if someone records using a standard SKU. Without being recorded, old or absent files for otherwise-skipped tests would cause some to fail. Not worth worrying about at this point, though. Seems only one cloud doesn't support premium SKUs at this time. If anyone were to record tests in that region, it would have to be against the public cloud at least.

Reply:
Why not make it non-recorded then?